File: 2020/036-JuliaVsPython/labQ.jl
using CPUTime

function execute()
	primes = []
	a = 100000000
	limit = 10001
	for j in a+1:2:a+10000
		prime = true
		for k in 3:2:limit
			if j % k == 0
				prime = false
				break
			end
		end	
		if prime
			push!(primes,j)
		end
	end
	println(length(primes))
end
@time @CPUtime execute()

File: 2020/036-JuliaVsPython/labQ.py
import time
import math

def execute():
	primes = []
	a = 100000000
	limit = 10001
	for j in range(a+1,a+10000,2):
		prime = True
		for k in range(3,limit,2):
			if j % k == 0:
				prime = False
				break
		if prime: primes.append(j)
	print(len(primes))

start = time.time()
execute()
print(time.time()-start)
